networkx/algorithms/components/attracting.py

```diff
@@ -11,7 +11,7 @@
 
 @not_implemented_for("undirected")
 def attracting_components(G):
-    """Generates the attracting components in `G`.
+    """Generates the attracting components in directed graph.
```

Comment:
Changes to the first line of the docstring as proposed in this PR are fine, though it's worth noting that the type of the graph (i.e. the fact that it's directed) is noted in the parameter description below, so arguably this is redundant.

I don't feel strongly about it though, but if they are switched we should double check that the new wording is grammatically correct (e.g. `Generates the attracting components in a directed graph`).